services/app-api/resolvers/submitDraftSubmission.ts

```diff
@@ -159,12 +160,27 @@ export function submitDraftSubmissionResolver(
 
         const updatedSubmission: StateSubmissionType = updateResult
 
-        // Send the email!
-        const emailData = emailer.generateCMSEmail(stateSubmission)
+        // Send CMS and state emails for submission!
+        const emailData = emailer.generateEmailTemplate({
+            template: 'CMS_NEW_PACKAGE',
+            submission: stateSubmission,
+        })
         const emailResult = await emailer.sendEmail(emailData)
```

Comment:
Agreed. See 604a96d . I left `sendEmail` on  the emailer because I thought it might be beneficial to expose the ability to invoke the email submit lambda directly,  and it also allows tests to mock that function, but using a single function now for cms and state emails each to generate the template and send (using `this.sendEmail`)